bootstrap/bootstrap.go

```diff
@@ -470,12 +527,22 @@ func setupStorage(
 				err,
 			)
 		}
+
+		err = batch.Commit(pebbleDB.Sync)
+		if err != nil {
+			return nil, fmt.Errorf("could not commit register updates: %w", err)
+		}
+
 		logger.Info().Msgf("database initialized with cadence height: %d", cadenceHeight)
 	}
+	//else {
+	//	// TODO(JanezP): verify storage account owner is correct
```

Comment:
I opened an issue for this: https://github.com/onflow/flow-evm-gateway/issues/651